Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/na-cordex-viz-cookbook-dev/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/share/miniconda3/envs/na-cordex-viz-cookbook-dev/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda3/envs/na-cordex-viz-cookbook-dev/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda3/envs/na-cordex-viz-cookbook-dev/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/share/miniconda3/envs/na-cordex-viz-cookbook-dev/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/usr/share/miniconda3/envs/na-cordex-viz-cookbook-dev/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/share/miniconda3/envs/na-cordex-viz-cookbook-dev/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
if USE_PBS_SCHEDULER:
    cluster = get_pbs_cluster()
else:
    cluster = get_gateway_cluster()
    
# Connect to cluster
from distributed import Client
client = Client(cluster)

# Display cluster dashboard URL
cluster
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[7], line 4[0m
[1;32m      2[0m     cluster [38;5;241m=[39m get_pbs_cluster()
[1;32m      3[0m [38;5;28;01melse[39;00m:
[0;32m----> 4[0m     cluster [38;5;241m=[39m [43mget_gateway_cluster[49m[43m([49m[43m)[49m
[1;32m      6[0m [38;5;66;03m# Connect to cluster[39;00m
[1;32m      7[0m [38;5;28;01mfrom[39;00m [38;5;21;01mdistributed[39;00m [38;5;28;01mimport[39;00m Client

Cell [0;32mIn[5], line 4[0m, in [0;36mget_gateway_cluster[0;34m()[0m
[1;32m      1[0m [38;5;28;01mdef[39;00m [38;5;21mget_gateway_cluster[39m():
[1;32m      2[0m [38;5;250m    [39m[38;5;124;03m""" Create cluster through dask_gateway[39;00m
[1;32m      3[0m [38;5;124;03m    """[39;00m
[0;32m----> 4[0m     [38;5;28;01mfrom[39;00m [38;5;21;01mdask_gateway[39;00m [38;5;28;01mimport[39;00m Gateway
[1;32m      6[0m     gateway [38;5;241m=[39m Gateway()
[1;32m      7[0m     cluster [38;5;241m=[39m gateway[38;5;241m.[39mnew_cluster()

[0;31mModuleNotFoundError[0m: No module named 'dask_gateway'

